linkerd/app/core/src/trace.rs

```diff
@@ -7,20 +7,32 @@ use tracing_subscriber::{
     reload, EnvFilter, FmtSubscriber,
 };
 
-const ENV_LOG: &str = "LINKERD2_PROXY_LOG";
-
-type Subscriber = Formatter<format::DefaultFields, format::Format<format::Full, Uptime>>;
+const ENV_LOG_LEVEL: &str = "LINKERD2_PROXY_LOG";
+const ENV_LOG_FORMAT: &str = "LINKERD2_PROXY_LOG_FORMAT";
 
 #[derive(Clone)]
-pub struct LevelHandle {
-    inner: reload::Handle<EnvFilter, Subscriber>,
+pub enum LevelHandle {
+    Json {
+        inner: reload::Handle<
+            EnvFilter,
+            Formatter<format::JsonFields, format::Format<format::Json, Uptime>>,
```

Comment:
Take it or leave it: I _might_ consider adding `JsonFormatter` and `PlainFormatter` type aliases for these, because this is a bit wordy; not a blocker though.

Reply:
💯 This would increase readability. By one confusion is wether these are Formatters or Subscribers, originally when there was only the Plain version it was type aliased as Subscriber. Going with the term Formatter now but please advise.